study.md

```diff
@@ -43,5 +48,8 @@ your deployed Ember app? What do you need if you want to use the `history` API
 instead of `hash` for `location`?
 
 ```md
-<!-- your answer here -->
+Hash URLs are URLs that contain a hashtag (#), which loads the starting state of the application and allows you to move around the site while keeping the app state in sync. Example:  #/posts/new  takes you to posts.new route.
+
+If you want to use the browser's history API, you need to make sure the Ember app is served from all URLs defined by your Router.map function.  If a user manually types the url (ex: /posts/new), you will need to configure your server to respond to it, since Ember will not do it for you.  You will also need to specify the location as 'history'.
```

Comment:
If you want to use history API, then you can't deploy on github, you would need deploy your ember app to Heroku. To host an Ember application on github, use ember build --environment production to build the project to dist/, because it is unable to run a server.